mllib-local/src/test/scala/org/apache/spark/ml/linalg/BLASSuite.scala

```diff
@@ -23,6 +23,15 @@ import org.apache.spark.ml.util.TestingUtils._
 
 class BLASSuite extends SparkMLFunSuite {
 
+  test("nativeL1Threshold") {
+    var vectorSize = 128
+    assert(getBLAS(vectorSize) == BLAS.f2jBLAS)
```

Comment:
minor: you don't need `var`. just `assert(getBLAS(128) == BLAS.f2jBLAS)` is sufficient and cleaner.

Reply:
UT updated to use constant directly.